app/controllers/projects_controller.rb

```diff
@@ -1,11 +1,25 @@
 class ProjectsController < ApplicationController
   before_action :ensure_logged_in, only: [:your_dependent_repos, :mute, :unmute,
                                           :unsubscribe, :sync]
-  before_action :find_project, only: [:show, :sourcerank, :about, :dependents,
-                                      :dependent_repos, :your_dependent_repos,
-                                      :versions, :tags, :mute, :unmute, :unsubscribe,
-                                      :sync, :score, :refresh_stats]
-  before_action :find_project_lite, only: [:top_dependent_repos, :top_dependent_projects]
```

Comment:
Is the performance of these two actions going to be a lot longer without the "lite" version of the query?

Reply:
It's really just the includes - I can't imagine there was ever any real need for `lite`

Reply:
I'll check the git logs though...

Reply:
I can't find any reasoning